client/widgets/TextControls.cpp

```diff
@@ -426,16 +428,19 @@ void CGStatusBar::init()
 
 void CGStatusBar::clickLeft(tribool down, bool previousState)
 {
-	if(!down && onClick)
+	if(!down && onClick )
 	{
-		onClick();
+		if(LOCPLINT && LOCPLINT->cingconsole->active)
```

Comment:
```suggestion
		if(LOCPLINT && LOCPLINT->cingconsole && LOCPINT->cingconsole->active)
```

Reply:
Pointless - LOCPLINT->cingconsole is created in costructor and deleted in destructor. Rest of the code does not checks for cingconsole either